src/Microsoft.AspNetCore.OData/Formatter/EdmLibHelpers.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using Microsoft.AspNet.OData.Interfaces;
+
+namespace Microsoft.AspNet.OData.Formatter
+{
+    internal static partial class EdmLibHelpers
```

Comment:
why do we need this partial? Cannot add into the original class?

Reply:
Partial classes are used in cases where a small bit of functionality is platform-specific and it's not easy to shim. This reason in this case is that how the assembly resolver is created and used.